livebook_examples/playing_mp3_file/playing_mp3_file.livemd

```diff
@@ -0,0 +1,89 @@
+# Playing MP3 File
+
+```elixir
+File.cd(__DIR__)
+Logger.configure(level: :error)
+
+Mix.install([
+  {:membrane_core, "~> 0.11.2"},
+  {:membrane_file_plugin, "~> 0.13.0"},
+  {:membrane_mp3_mad_plugin, "~> 0.14.0"},
+  {:membrane_ffmpeg_swresample_plugin, "~> 0.16.1"},
+  {:membrane_aac_fdk_plugin, "~> 0.14.0"},
+  {:membrane_kino_plugin, github: "membraneframework-labs/membrane_kino_plugin"}
+])
+```
+
+## Installation
+
+To run this demo one needs to install native dependencies:
+
+1. [MP3 MAD](https://github.com/membraneframework/membrane_mp3_mad_plugin/tree/v0.14.0#installation)
+2. [AAC FDK](https://github.com/membraneframework/membrane_aac_fdk_plugin#installation)
+3. [SWResample FFmpeg](https://github.com/membraneframework/membrane_ffmpeg_swresample_plugin#installation)
```

Comment:
I think that the `Installation` section should appear before the dependencies setup. This way, the user should be aware, of why something went wrong.  Moreover, the mad plugin has a poor description in the installation section, so to improve experience of this notebook users, we should either list here all required commands for installing native dependencies (on Mac and Linux) or/and create a PR improving mad plugin readme.

Reply:
I am afraid it is obligatory to put the dependencies setup at the beginning of the livebook. There is no possibility of even putting the markdown cell before it ;/

Reply:
Ok